src/java_bytecode/java_bytecode_convert_method.cpp

```diff
@@ -81,7 +87,7 @@ class java_bytecode_convert_methodt:public messaget
     symbol_exprt symbol_expr;
     size_t start_pc;
     size_t length;
-    bool is_parameter;
+    bool is_parameter=false;
```

Comment:
Well, indeed non of the POD members will be initialised, i.e., start_pc, length would also benefit from initialisation. Would you mind adding a default constructor that takes care of all of them, i.e., `():start_pc(0), length(0), is_parameter(false) {}`.